contracts/tokens/SecurityToken.sol

```diff
@@ -1061,9 +1057,9 @@ contract SecurityToken is ERC20, ReentrancyGuard, SecurityTokenStorage, IERC1594
 
     function updateFromRegistry() public {
         _onlyOwner();
-        moduleRegistry = PolymathRegistry(polymathRegistry).getAddress("ModuleRegistry");
-        securityTokenRegistry = PolymathRegistry(polymathRegistry).getAddress("SecurityTokenRegistry");
-        polyToken = PolymathRegistry(polymathRegistry).getAddress("PolyToken");
+        moduleRegistry = IModuleRegistry(IPolymathRegistry(polymathRegistry).getAddress("ModuleRegistry"));
```

Comment:
might be unnecessary to cast the registry to `IPolymathRegistry` as it is already declared as `IPolymathRegistry` in `SecurityTokenStorage`.

same in `UpgradeableModuleFactory::76` (not part of this PR) where `polymathRegistry` is a statevar inherited from `ModuleFactory` declared as `IPolymathRegistry` already